_pages/ato/types.md

```diff
@@ -3,11 +3,11 @@ title: Types of ATO
 navtitle: Types
 ---
 
-In most cases, the types of ATO that will be pursued for GSA systems are the *GSA Lightweight ATO (LATO)*. The GSA LATO process is described in a guide on [Insite](https://insite.gsa.gov/portal/content/627230) (search for "Lightweight Security Authorization Process" on that page). Systems that are under development can be operated without an ATO provided they fulfill the requirements for [pre-assessment for internal government use](#conditions-for-pre-assessment).
+In most cases, the types of ATO that will be pursued for GSA systems are the *GSA Lightweight ATO (LATO)*. The GSA LATO process is described in a guide on [Insite](https://insite.gsa.gov/portal/content/627230) (search for "Lightweight Security Authorization Process" on that page). Systems that are under development must fulfill the requirements for [pre-assessment for internal government use](#conditions-for-pre-assessment).
```

Comment:
> Systems that are under development must fulfill...

The distinguishing characteristic is whether it has an ATO yet or not, not whether it is under development.